app/templates/index.html

```diff
@@ -1,116 +1,107 @@
-			      	</label>
-			    </div>
-			    <div class="checkbox">
-			      	<label for="pdf">
-			      		<input type="checkbox" id="pdf" name="pdf" checked="checked">PDF
-			      	</label>
-			  	</div>
-			  	<div class="form-group">
-			  		<label for="inputFile">Upload a CSV file to generate badges</label>
-			  		<input type="file" id="inputFile" name="file">
-			  	</div>
-			  	<div class="text-center"><strong>OR</strong></div>
-			  	<div class="form-group">
-			  		<label for="textArea">Enter data manually</label>
-					<textarea class="form-control" id="textArea" name="csv"></textarea>
-			  	</div>
-			  	<div class="form-group">
-						<div class="text-center guide">
-                 Check out our <a href="/guide" target="_blank">User Input Guide</a>
-						</div>
-			  		<label for="textField">Enter Eventyay URL and Type</label>
-					<input type="text" class="form-control" id="textField" name="eventyay_url">
-			  	</div>
-			  	<div class="form-group">
-			  		<label for="imageFile">
-			  			Upload a PNG file for badge background
-			  		</label>
-					<input type="file" id="imageFile" name="image">
-			  	</div>
-			  	<div class="text-center"><strong>OR</strong></div>
-			  	<div class="form-group">
-			  		<label for="inputFile">
-			  			Select from default images
-			  		</label>
-			  		<div class="btn-group">
-						<button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-							<span class="placeholder">
-								Select a background
-							</span>
-							<span class="glyphicon glyphicon-chevron-down"></span>
-						</button>
-						<ul class="dropdown-menu">
-							{% for i in default_background %}
-							<li class="menu-options" data-item="{{i}}">
-								<img src="static/uploads/{{i}}"> {{i}}
-							</li>
-							{% endfor %}
-						</ul>
-					</div>
-					<label for="inputFile">
-			  			Enter custom text for badges
-			  		</label>
-					<input id="image_text" type="text" class="form-control" placeholder="The text entered here will appear on badges" name="text_on_image"/>
-					<input type="hidden" name="img-default" value="">
-				</div>
-				<div class="text-center"><strong>OR</strong></div>
-				<div class="form-group">
-					<label for="inputFile">
-			  			Select a background color
-			  		</label>
-					<input type="text" id="picker" class="form-control" value="#4fc8db" name="bg_color">
-				</div>
 
-			  	<div class="form-group">
-			  		<label for="inputFile">
-			  			<span class="label label-info">OPTIONAL</span>
-			  			Upload a config file (JSON) to customise badges
-			  		</label>
-					<input type="file" id="inputFile" name="config">
-			  	</div>
-				<button type="submit" class="btn btn-primary"><span class="glyphicon glyphicon-upload"></span>  Upload</button>
-			</form>
-		</div>
-	</div>
+  	<form  action="{{ url_for('upload') }}"  method="post" enctype="multipart/form-data" >
```

Comment:
This file lacks indentation. Please fix this. Please read more about it [here](https://www.w3schools.com/html/html5_syntax.asp)